app/src/main/res/layout/navdrawer.xml

```diff
@@ -22,6 +22,7 @@
   android:layout_height="match_parent"
   android:orientation="vertical"
   android:layout_gravity="start"
+  android:clickable="true"
```

Comment:
What does this do? Does it fix the problem where we were able to click things underneath?